packages/components/src/tools-panel/tools-panel/hook.ts

```diff
@@ -285,20 +301,25 @@ export function useToolsPanel(
 			menuItems,
 			panelId,
 			registerPanelItem,
+			registerResetAllFilter,
+			resetAllFilters,
```

Comment:
What is the reason for adding `resetAllFilters` to the context?

Reply:
Yeah, I couldn't see either why it would need to be exposed in the context, unless to introspect it during testing? 🤔